2020/summer_meeting/session_three_logging_and_observability.md

```diff
@@ -0,0 +1,103 @@
+#Session 3 - Logging / Observability
```

Comment:
I think you need spaces after all of these hashes in order for markdown to do the right thing